tests/parser/EclipseGridTests.cpp

```diff
@@ -40,9 +42,24 @@
 #include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
 #include <opm/parser/eclipse/EclipseState/Grid/EclipseGrid.hpp>
 #include <opm/parser/eclipse/EclipseState/Grid/GridDims.hpp>
+#include <opm/parser/eclipse/EclipseState/Grid/NNC.hpp>
 
 #include <opm/parser/eclipse/Parser/Parser.hpp>
 
+#include <opm/io/eclipse/EclFile.hpp>
+
+std::string getCurrentWorkingDir() {
+
+    char buff[PATH_MAX];
+
+    if (getcwd( buff, PATH_MAX )==NULL) {
```

Comment:
yes we can. Replacing function getCurrentWorkingDir() with the following

```
std::string cwd = boost::filesystem::current_path().c_str();
```

Reply:
Actually you should not need to add `.c_str()` (or `.native()` or `.string()`), since path has an implicit conversion to string (I think). In any case they should all do the same.